simnode/generator.py

```diff
@@ -108,27 +233,35 @@ def _calculate_parallel_coords(offset: float, line_width: float) \
 
                 # Draw line separating left from right lanes
                 if road.left_lanes > 0 and road.right_lanes > 0:
-                    offset = initial_line_offsets[road.left_lanes]
+                    divider_rel_off = -0.5 + road.left_lanes/(road.left_lanes+road.right_lanes)
+                    offset_list_divider = _calculate_offset_list(relative_offset=divider_rel_off,
+                                                                 absolute_offset=-side_line_offset)
                     left_right_divider = Road("line_yellow_double", rid=road.rid + "_left_right_divider")
-                    left_right_divider_nodes \
-                        = _calculate_parallel_coords(offset, 2 * self.add_roads_to_scenario.line_width)
+                    left_right_divider_nodes = \
+                        _calculate_parallel_coords_list(offset_list_divider, 2 * self.add_roads_to_scenario.line_width)
```

Comment:
Is this a change made by your formatter?

Reply:
I am not sure, I will move the '=' back down